infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/utils/LogReplicationConfigManager.java

```diff
@@ -148,7 +152,7 @@ private void init() {
         registeredSubscribers.add(getDefaultSubscriber());
         // TODO (V2): This builder should be removed after the rpc stream is added for Sink side session creation.
         //  and logical group subscribers should come from client registration.
-        registeredSubscribers.add(getDefaultLogicalGroupSubscriber());
+        //registeredSubscribers.add(getDefaultLogicalGroupSubscriber());
```

Comment:
Wouldn't this be addressed as part of this PR?

Reply:
This layer, for the source side is fine. 
But for the sink side, we'll need to add the session that the sink receives via the new RPC